src/main/java/lesson_jdbc_01/OrderDaoImpl.java

```diff
@@ -14,15 +14,58 @@
 	public Set<Order> getAllOrders() throws SQLException {
 		Set<Order> orders = new HashSet<>();
 		Connection conn = ConnectToDB.getConnection();
-		PreparedStatement stmt = conn.prepareStatement("select * orders");
+		PreparedStatement stmt = conn.prepareStatement("SELECT * FROM orders");
+		ResultSet rsOrder = stmt.executeQuery();
+		while (rsOrder.next()) {
+			Order order = new Order(rsOrder.getBigDecimal("order_num"), null, rsOrder.getDate("order_date"),
+					rsOrder.getString("mfr"), rsOrder.getBigDecimal("qty"), rsOrder.getBigDecimal("amount"));
+
+			stmt = conn.prepareStatement("SELECT * FROM products WHERE product_id=?");
+			stmt.setString(1, rsOrder.getString("product"));
+			ResultSet rsProduct = stmt.executeQuery();
+			while (rsProduct.next()) {
+				Product product = new Product(rsProduct.getString("product_Id"), rsProduct.getString("mfr_Id"),
+						rsProduct.getString("description"), rsProduct.getBigDecimal("price"),
+						rsProduct.getBigDecimal("qty_On_Hand"));
+				order.setProduct(product);
+			}
+			rsProduct.close();
+			orders.add(order);
+		}
+
+		rsOrder.close();
+		stmt.close();
+		conn.close();
+		return orders;
+	}
+
+	@Override
+	public Set<Order> getAllOrdersJoin() throws SQLException {
+		Set<Order> orders = new HashSet<>();
+		Connection conn = ConnectToDB.getConnection();
+		PreparedStatement stmt = conn.prepareStatement(
+				"SELECT orders.order_num, orders.order_date, orders.mfr, products.product_id, products.mfr_id, "
```

Comment:
maybe better to use "select*" instead of writing all needed fields to make sql query easier